charts/traefik/values.webinternal.yaml.gotmpl

```diff
@@ -0,0 +1,12 @@
+ports:
+  webinternal:
+    nodePort: 32081
+    port: 8001
+    expose:
+      default: true
+    exposedPort: 81
+    protocol: TCP
+    http3:
+      enabled: false
```

Comment:
is http2 enabled?

Reply:
I think it is by default https://github.com/traefik/traefik/issues/139